en/deploy-on-aws-eks.md

```diff
@@ -32,8 +32,8 @@ To verify whether AWS CLI is configured correctly, run the `aws configure list`
 
 - Instance types: to gain better performance, the following is recommended:
     - PD nodes: `c5.xlarge`
-    - TiDB nodes: `c5.2xlarge`
-    - TiKV or TiFlash nodes: `r5b.2xlarge`
+    - TiDB nodes: `c5.4xlarge`
+    - TiKV or TiFlash nodes: `m5.4xlarge`
 - Storage: Because AWS supports the [EBS `gp3`](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volume-types.html#gp3-ebs-volume-type) volume type, it is recommended to use EBS `gp3`. For `gp3` provisioning, the following is recommended:
```

Comment:
```suggestion
- Storage: Because AWS supports the [EBS `gp3`](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/general-purpose.html#gp3-ebs-volume-type) volume type, it is recommended to use EBS `gp3`. For `gp3` provisioning, the following is recommended:
```